packages/loader/container-loader/src/deltaQueue.ts

```diff
@@ -12,14 +12,9 @@ export class DeltaQueue<T> extends TypedEventEmitter<IDeltaQueueEvents<T>> imple
     private readonly q = new Deque<T>();
 
     /**
-     * Tracks whether the system has requested the queue be paused.
+     * Tracks the number of pause requests for the queue
      */
-    private sysPause = 1;
-
-    /**
-     * Tracks whether the user of the container has requested the queue be paused.
-     */
-    private userPause = 0;
+    private pauseCount = 0;
```

Comment:
Change in behavior? It should be 1 if we do not change any of existing pause/resume semantics.

Reply:
you are right.